content/en/docs/setup/on-premises-metal/krib.md

```diff
@@ -0,0 +1,95 @@
+
+{{< note >}}
+**Note:** KRIB is not a _stand-alone_ installer: Digital Rebar templates drive a standard *[kubeadm](/docs/admin/kubeadm/)* configuration that manages the Kubernetes installation with the [Digital Rebar cluster pattern](https://provision.readthedocs.io/en/tip/doc/arch/cluster.html#rs-cluster-pattern) to elect leaders _without external supervision_.
+{{< /note >}}
+
+
+KRIB features:
+
+* zero-touch, self-configuring cluster without pre-configuration or inventory
+* very fast, no-ssh required automation
+* bare metal, on-premises focused platform
+* highly available cluster options (including splitting etcd from the controllers)
+* dynamic generation of a TLS infrastructure
+* composable attributes and automatic detection of hardware by profile
+* options for persistent, immutable and image-based deployments
+* support for Ubuntu 18.04, CentOS/RHEL 7 and others
+
+## Creating a cluster
+
+Review [Digital Rebar documentation](https://https://provision.readthedocs.io/en/tip/README.html) for details about installing the platform.
+
+The Digital Rebar Provision Golang binary is installed on a Linux-like system with 16 Gb of RAM or larger (Packet.net Tiny and Rasberry Pi are also acceptable).
+
+### (1/5) Discover servers
+
+Following the [Digital Rebar installation](https://provision.readthedocs.io/en/tip/doc/quickstart.html), allow one or more servers to boot through the _Sledgehammer_ discovery process to register with the API.  This will automatically install the Digital Rebar runner and to allow for next steps.
+
+### (2/5) Install KRIB Content and Certificate Plugin
+
+Upload the KRIB Content bundle (or build from [source](https://github.com/digitalrebar/provision-content/tree/master/krib)) and the Cert Plugin for your DRP platform (e.g.: [amd64 Linux v2.4.0](https://s3-us-west-2.amazonaws.com/rebar-catalog/certs/v2.4.0-0-02301d35f9f664d6c81d904c92a9c81d3fd41d2c/amd64/linux/certs)).  Both are freely available via the [RackN UX](https://portal.rackn.io).
+
+### (3/5) Start your cluster deployment
+
+{{< note >}}
+**Note:** KRIB documentation is dynamically generated from the source and will be more up to date than this guide.
+{{< /note >}}
+
+Following the [KRIB documentation](https://provision.readthedocs.io/en/tip/doc/content-packages/krib.html), create a Profile for your cluster and assign your target servers into the cluster Profile.  The Profile must set `krib\cluster-name` and `etcd\cluster-name` Params to be the name of the Profile.  Cluster configuration choices can be made by adding additional Params to the Profile; however, safe defaults are provided for all Params.
```

Comment:
is `Param` an object type?
possibly change to `parameter`?

Reply:
Param is an API type for Digital Rebar